drivers/hmis/app/graphql/types/hmis_schema/has_clients.rb

```diff
@@ -22,8 +22,14 @@ def clients_field(name = :clients, description = nil, type: [Types::HmisSchema::
         end
       end
 
-      def resolve_clients(scope = object.clients, sort_order: :last_name_asc, _user: current_user, no_sort: false)
-        clients_scope = scope
+      def resolve_clients_with_loader(association_name = :clients, sort_order: :last_name_asc, user: current_user, no_sort: false)
```

Comment:
How does this method get called? From the client search it calls `resolve_clients`

Reply:
The loader we're currently using loads an association for an object, so it would be called within an object type. For example, `Organization.projects` would use `resolve_projects_with_loader`. Root queries don't have a base object, so there's no association to load. So we just use `resolve_*` at that level.

Reply:
Okay I see. So `resolve_clients_with_loader` isn't being used at all, because we don't load clients as an association anywhere in the schema (yet). Is that right?

Also by that logic, shouldn't we be using `resolve_enrollments_with_loader` on the Client?

I don't love that we are repeating the sort/filter logic twice everywhere (once in normal resolve, once in resolve_with_loader). If this code is going to change frequently, maybe we should pull it out into a helper function, or else we're bound to forget to keep each pair up-to-date. IIUC they always should match, we don't want different behavior for each.

Reply:
1. Yes, that's right. It's there for when we need it.
2. Because enrollments are paginated, there's no benefit to using the loader, since it still has to do a DB query for each paginated set. I think using the loader actually causes the request to do **more** DB queries than not using the loader, because it will try to preload the enrollments, then do the paginated queries on top of that.
3. Yeah, it's probably wise to put all that into a helper. I can take a crack at that.

Reply:
Added the helper functions